### ISSUE TITLE
Issue with coherent scattering

### ISSUE THREAD

**Issue by mcolomermolla** [SNEWS2/snewpy#295]
Trying to simulate the DS20k detector I encountered the following problems:

- Material is not defined for ds20 detector (that's not a real problem because one can give it directly to rate calculator), but maybe for the user it is not obvious.
- The coherent scattering channels are dropped as they have a different binning compared to others:
`
# drop coherent scattering channels, which have different binning
                df = df[df["name"].str.startswith('coh_') == False]
`

**Comment by mcolomermolla** [comment]
> Thanks for writing these up!
> 
> I’ll start with your last two points:
> 
> * It looks like the efficiency files (e.g. [`effic_coh_helm_nue_Ar_ds20.dat`](https://github.com/SNOwGLoBES/snowglobes/blob/master/effic/ds20/effic_coh_helm_nue_Ar_ds20.dat) give an efficiency of `0.0` independent of energy; I suspect that’s why your rates are zero even after the modifications.
> * Yes—`snowglobes_data` only contains the efficiency files present in the SNOwGLoBES repo. In practice, I don’t think these missing files matter much, though: Since the visible energy in neutrino-electron scattering is O(MeV)—i.e. several orders of magnitude higher than typical signals these DM detectors are built for—I’d expect that the efficiency is close to 1, which is also the default value used by snewpy if no efficiency file is provided.
> 
> Once those two points are fixed (which are outside the scope of snewpy, really; probably something we should discuss with DS20k folks in SNEWS first, and then discuss/track in an issue on the SNOwGLoBES repo), I agree the first two points are things we should improve in snewpy.

Yes, I realised after that efficiencies are zero... so basically, deleting those files so that it uses by default eff=1 should be fine for the moment to get a reasonable output for the coherent channels. However, it still won't run all the channels together because of the different binning, I will have a look at that.

**Comment by OliverMacfadyen** [comment]
Hi I am a PhD student on DarkSide and I ave just started working on snewpy and snews with a group at Oxford university. I was just wondering if anyone would be available to get me up to speed on this issue, where it is currently up to and if there is anything I can do to help.

**Comment by mcolomermolla** [comment]
Hi Oliver,
Sorry for the late reply. At the moment, there have been no updates. If you want, I can write you an email and we set a zoom meeting so that I can explain to you the issues and where to start as we originally planned.

**Comment by Sheshuk** [comment]
Fixed in #319
